Makefile

```diff
@@ -8,3 +8,15 @@ DOCKER_RUN=docker run -v $(CURDIR):/mnt otelopscol-build
 .PHONY: package-googet
 package-googet:
 	$(DOCKER_RUN) /bin/bash -c "cd /mnt; make -f .build/Makefile VERSION=$(VERSION) ARCH=$(ARCH) package-googet"
+
+# gcp collector related variables
+IMAGE_NAME=gcpcol_image
+CONTAINER_NAME=gcpcol_container
+
+.PHONY: gcpcol
+gcpcol:
+	docker build -t $(IMAGE_NAME) ./linux
+	docker run -i --name=$(CONTAINER_NAME) $(IMAGE_NAME)
+	docker cp $(CONTAINER_NAME):/go/gcp-otel.tar.gz .
```

Comment:
Interesting, I guess this works just as well given your setup, but I would normally mount your local directory as a volume in the docker container instead.

(see the `-v $(CURDIR):/mnt` flag above)

That way you can run local scripts in the docker container and any files generated in the container (inside the mounted volume) will exist on your local system as well.

Reply:
perfect